mailcatcher/README.md

```diff
@@ -6,3 +6,30 @@ FROM quay.io/continuouspipe/mailcatcher:stable
 
 Port 1025 is where mail should be relayed to
 Port 1080 is where you can visit the UI
+
+Example `docker-compose` configuration:
+
+```
+mailcatcher:
+    image: quay.io/continuouspipe/mailcatcher:stable
+    extra_hosts:
+        - "mailcatcher:127.0.0.1"
+    expose:
+        - 25
+    ports:
+        - "1080:80"
```

Comment:
well, those don't work :-)
from the log:
```
mailcatcher_1  | Starting MailCatcher
mailcatcher_1  | ==> smtp://0.0.0.0:25
mailcatcher_1  | ==> http://0.0.0.0:80
```
Once they do I'll be happy to update the usage doc :)